website/docs/components/side-panel/side-panel.md

```diff
@@ -94,8 +90,8 @@ Table: SidePanel.Header styles
 
 | Appearance example     | Styles         |
 | ---------------------- | -------------- |
-| ![](static/header.png) | `height: 52px`, `padding-bottom: var(--spacing-1x)`, `align-items: center`, `border-bottom: 1px solid var(–-border-primary)` |
-| ![](static/closeIcon.png)   | The `Close` icon has an L size and uses `--icon-secondary-neutral` token for color. The click zone of the icon is `48px * 48px`.   |
+| ![](static/header.png) | `height: 52px`, `align-items: center`, <br>`padding-top: var(--spacing-1x)`, <br>`padding-bottom: var(--spacing-1x)`, <br>`border-bottom: 1px solid var(–-border-primary)` |
+| ![](static/closeIcon.png)   | The **Close** button has an L size and an L size icon inside. The icon uses `--icon-secondary-neutral` token for color. The click zone of the button is `48px * 48px`.   |
```

Comment:
Maybe we should rewrite it to something like this:

The **Close** button uses a button with size L and an L sized icon inside.

I think we do not need to describe the styles of the elements of the other components here. If we change something in the Button, we will need to change it here too, and we might miss it.

Reply:
changed the wording